ui/jobs/jobs.js

```diff
@@ -169,6 +169,14 @@ const kAbility = {
   Dia: '4094',
   Assize: 'DF3',
 };
+// TODO: Some of them are missed in effect_id.js,
+// if effect_id.js is fixed, this can be removed.
+const brdDoTs = {
```

Comment:
Can you put these in the `known_mapping` section of `util/gen_effect_id.py` and re-run that script rather than including them manually here? You need to use the decimal numbers there, rather than the hex values you have here.

Reply:
Done.